### ISSUE TITLE
Just says "Sorry, this content could not be embedded."

### ISSUE THREAD

**Issue by numbersixnz** [leewillis77/wp-github-oembed#36]
As per title, just says "Sorry, this content could not be embedded" on Wordpress 6.0.  I assume WP6 isn't supported and that's fine, but FYI.  Unsure if this is an issue with the plugin or with WP6 actually but based on the plugin docs I should be able to just paste a repo URL into a post and have it pull info from GitHub.  That doesn't happen.

**Comment by leewillis77** [comment]
There's no reason it shouldn't work on WordPress 6.0 - I've just tested on a fresh install, and everything works as expected:
https://capture.dropbox.com/CRMS16HZmKDUZaVt

It's likely something unique to your setup blocking the request (SSL problem / security plugin / maintenance mode plugin or similar).

**Comment by digitalformula** [comment]
> There's no reason it shouldn't work on WordPress 6.0 - I've just tested on a fresh install, and everything works as expected: https://capture.dropbox.com/CRMS16HZmKDUZaVt
> 
> It's likely something unique to your setup blocking the request (SSL problem / security plugin / maintenance mode plugin or similar).

Interesting, I did the same and it doesn't work on a new install for me either.  I'm using [this repo](https://github.com/leewillis77/wp-github-oembed) as my test.

Results in WP admin:

![image](https://user-images.githubusercontent.com/423229/177675356-8ee24f38-6c3a-433a-814e-1142d78f9617.png)

Browser console:

![image](https://user-images.githubusercontent.com/423229/177675620-9255338c-2569-4584-b15e-68c465a6dca6.png)

Visiting the URL in the console returns this:

```{"code":"rest_missing_callback_param","message":"Missing parameter(s): url","data":{"status":400,"params":["url"]}}```

This site has GitHub Embed enabled and nothing else.  WP6.0, TwentyTwentyTwo theme.

**Comment by leewillis77** [comment]
As I said, we've confirmed that it works here on WordPress 6.0 and I've not seen any other reports of people having problems, so it must come down to something on your specific setup. If you've ruled out other plugins you might also want to look at network setup on the box, dns checks, firewall rules or similar, or your browser config (ad-blocking or similar addons maybe?). 

Either way, there doesn't appear to be an issue with the extension itself.

**Comment by digitalformula** [comment]
> As I said, we've confirmed that it works here on WordPress 6.0 and I've not seen any other reports of people having problems, so it must come down to something on your specific setup. If you've ruled out other plugins you might also want to look at network setup on the box, dns checks, firewall rules or similar, or your browser config (ad-blocking or similar addons maybe?).
> 
> Either way, there doesn't appear to be an issue with the extension itself.

Yes and as *I* said, I've tested it the same way and it doesn't work for me.  Since we're talking details and supposed issues with my setup(s), here's what I've tested.

- Two different hosts (DigitalOcean and local)
- The actual website I did want to use this on + a brand new empty WP 6.0 install on DigitalOcean (new install has literally nothing enabled except this plugin)
- Neither new install nor production instance a firewall installed
- Chrome, Brave, Firefox
- Windows, Mac, Linux
- VPN connected & disconnected

That's pretty thorough no matter how you look at it.

Either way, it's not a big issue as there are alternatives to this plugin.  I simply thought the author might want to be aware.  :man_shrugging:
